nixos/modules/services/networking/nebula.nix

```diff
@@ -210,6 +210,11 @@ in
             ''
             settings
           );
+        capabilities = concatStringsSep " " ([
+          "CAP_NET_ADMIN"
+        ] ++ (optionals (settings.lighthouse.serve_dns or false) [
```

Comment:
This condition should probably be:

```
let
  nebulaPort = settings.listen.port;
  dnsPort = if (settings.lighthouse.serve_dns or false) then settings.lighthouse.dns.port or -1 else -1;
in nebulaPort > 0 && nebulaPort < 1024 || dnsPort > 0 && dnsPort < 1024;
```

Do you mind adding `services.nebula.networks.<name>.dns.{enable,port,host}` as well? The port should likely default to 5353 so we don't need the extra capability.